examples/linux/golioth_basics/main.c

```diff
@@ -2,31 +2,22 @@
 #include "golioth_basics.h"
 #include <string.h>
 #include <assert.h>
+#include <stdlib.h>
 
 #define TAG "main"
 
-// Include user's Golioth credentials.
-//
-// This is a required file, so if it doesn't exist, you will
-// need to create it and add the following:
-//
-//    #define GOLIOTH_PSK_ID "device@project"
-//    #define GOLIOTH_PSK "secret"
-//
-// The Golioth credentials can be found at:
-//
-//    console.golioth.io -> Devices -> (yourdevice) -> Credentials
-#include "credentials.inc"
-
 int main(void) {
+    char* golioth_psk_id = getenv("GOLIOTH_PSK_ID");
+    char* golioth_psk = getenv("GOLIOTH_PSK");
```

Comment:
Can we add an error message if either of these don't exist in the environment?

Reply:
Good call, will do!

Reply:
@sam-golioth updated, let me know if there are any issues with error message style 👍🏻